webui/tests/selenium/webui-selenium-test.py

```diff
@@ -443,6 +437,12 @@ def get_env():
     sleeptime = os.environ.get('BAREOS_DELAY')
     if sleeptime:
         SeleniumTest.sleeptime = float(sleeptime)
+    if(os.environ.get('TRAVIS') == 'true'):
+            SeleniumTest.browser = 'none'
+            SeleniumTest.username = 'citest'
+            SeleniumTest.password = 'citestpass'
+            SeleniumTest.sauce_username = os.environ.get('SAUCE_USERNAME')
+            SeleniumTest.access_key = os.environ.get('SAUCE_ACCESS_KEY')
```

Comment:
Set the test variables different if we are currently in a travis build.
`SAUCE_USERNAME` and `SAUCE_ACCESS_KEY` need to be set in Travis' build settings so that they will not be exposed.